src/Frontend/Themes/Bootstrap4/Modules/Profiles/Layout/Templates/Login.html.twig

```diff
@@ -1,6 +1,8 @@
 {% import 'Core/Layout/Templates/Alerts.html.twig' as alerts %}
 
 <section id="loginForm" class="profiles">
+  <h1 class="mb-2">{{ 'lbl.Login'|trans|ucfirst }}</h1>
```

Comment:
Nieuwe vertaling toegevoegd in installer?

Reply:
zit er al in